deno-bootstrap/index.ts

```diff
@@ -44,25 +42,32 @@ const server = Deno.serve(
     // Restore host and connection headers.
     req.headers.delete("host");
     req.headers.delete("connection");
-    if (req.headers.has("X-Deno-Worker-Host"))
+    if (req.headers.has("X-Deno-Worker-Host")) {
       req.headers.set("host", req.headers.get("X-Deno-Worker-Host")!);
-    if (req.headers.has("X-Deno-Worker-Connection"))
+    }
+    if (req.headers.has("X-Deno-Worker-Connection")) {
       req.headers.set(
         "connection",
-        req.headers.get("X-Deno-Worker-Connection")!
+        req.headers.get("X-Deno-Worker-Connection")!,
       );
+    }
 
     req.headers.delete("X-Deno-Worker-URL");
     req.headers.delete("X-Deno-Worker-Host");
     req.headers.delete("X-Deno-Worker-Connection");
     return mod.default.fetch(req);
-  }
+  },
 );
 
-globalThis.onerror = (e) => {
+addEventListener("error", (e) => {
   console.error(e.error);
   e.preventDefault();
-};
+});
+
+addEventListener("unhandledrejection", (e) => {
+  console.error(e.reason);
+  e.preventDefault();
+});
```

Comment:
fyi @stainless-em we ended up removing this in a recent version. we rely on this throwing in val.town. hopefully it's easy enough to add-back downstream